contracts/MerklePatriciaProofMock.sol

```diff
@@ -0,0 +1,46 @@
+pragma solidity ^0.4.23;
+
+import "./MerklePatriciaProof.sol";
+import "./util.sol";
+
+/**
+ *	@title This contract is an mock contract for MerklePatriciaProof library contract.
+ *
+ *	@notice It passes the parameters which it receives from test cases.
+ *
+ */
+contract MerklePatriciaProofMock is Util{
+
+    function MerklePatriciaProofMock(){
+
+    }
+
+    /**
+     * @dev This method is used for Account proof
+     * @param value The terminating value in the trie.
+     * @param encodedPath The path in the trie leading to value.
+     * @param rlpParentNodes The rlp encoded stack of nodes.
+     * @param root The root hash of the trie.
+     * @return The boolean validity of the proof.
+     */
+    function verifyAccount(bytes32 value, bytes encodedPath, bytes rlpParentNodes, bytes32 root) returns (bool){
+
```

Comment:
method parameters should be in new line.
you can refer OpenSTUtility.sol and Core.sol for reference.
Also returns can be in new line.